doc/api/index.rst

```diff
@@ -69,4 +69,8 @@ for reading/writing of data files:
   * `Kikuchipy <https://kikuchipy.org>`_: Processing, simulating and analyzing
     electron backscatter diffraction (EBSD) patterns in Python 
   * `PyXem <https://pyxem.readthedocs.io>`_: An open-source Python library for
-    multi-dimensional diffraction microscopy. 
+    multi-dimensional diffraction microscopy.
+  * `exSpy <https://hyperspy.org/exspy/>`_: Analysis of X-rays Energy Dispersive
```

Comment:
As both are used and valid, I simply removed the "s". I usually prefer the X-ray EDS because it is consistent with EELS and it is consistent with our use of EDS, as opposed to EDX.

Reply:
Well, I have seen either of the abbreviations EDS/EDX for "Energy Dispersive X-ray Spectroscopy" (11.5 M hits on google) and had not come across putting the "X-ray" first so far (125k hits on google) .

Reply:
This is used in the Williams & Carter book (but also in reference paper by Zaluzec or Watanabe), which I use most of the times as reference for this topic and I guess this is why I tend to use this wording.